_components/link/guidance/usability-should.md

```diff
@@ -14,6 +14,8 @@
 
 - **If you use an external link indicator, use it consistently for all text links.** If your project uses an external link indicator (like an icon), use it for all text links across your site. If users learn to associate an external link with the indicator, they will also expect that text links without an indicator are not external links. Icon- or image-only links like social share buttons or logos do not need the same treatment as text links.
 
+- **Use `rel="noreferrer"` property on external links.** Setting `noreferrer` on links prevents the browser leaking referrer information.
```

Comment:
Optional:
This works but I had some questions and I wonder if users might also have them:
Do we recommend this on every external link? Is there a time when users might want to exclude this? Will users understand what "leaking referrer information" means?

Reply:
Re: "leaking referrer information" — I'd say no. This needs a bit more plain-language translation

Reply:
Good points, updated in 43fe347. It now reads:

>Setting `noreferrer` on links prevents the browser leaking information about the original web address.

_Based on this [MDN guidance on referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer#:~:text=When%20you%20click%20a%20link%2C%20the%20Referer%20contains%20the%20address%20of%20the%20page%20that%20includes%20the%20link.%20When%20you%20make%20resource%20requests%20to%20another%20domain%2C%20the%20Referer%20contains%20the%20address%20of%20the%20page%20that%20uses%20the%20requested%20resource.)._